delegation-toolkit/concepts/delegation/index.md

```diff
@@ -0,0 +1,185 @@
+---
+description: Learn about delegation, the delegation lifecycle, and the Delegation Framework.
+sidebar_position: 2
+toc_max_heading_level: 2
+---
+
+import Tabs from "@theme/Tabs"; 
+import TabItem from "@theme/TabItem";
+
+# Delegation
+
+*Delegation* is the ability for a [MetaMask smart account](../smart-accounts.md) to grant permission to another smart account
+or externally owned account (EOA) to perform specific executions on its behalf.
+The account that grants the permission is called the *delegator account*, while the account that receives the permission
+is called the *delegate account*.
+
+The MetaMask Delegation Toolkit follows the [ERC-7710](https://eips.ethereum.org/EIPS/eip-7710) standard for smart contract delegation.
+In addition, users can use [caveat enforcers](caveat-enforcers.md) to apply rules and restrictions to delegations.
+For example: Alice delegates the ability to spend her USDC to Bob, limiting the amount to 100 USDC.
+
+## Delegation lifecycle
+
+The delegation lifecycle is as follows:
+
+1. **Create a delegation** - The delegator account creates and signs a delegation.
+
+2. **Apply caveats** - The caveats applied to the delegation specify conditions under which
+   the delegation can be redeemed.
+
+3. **Store the delegation** - A dapp can store the delegation, enabling retrieval for future redemption.
+
+4. **Redeem the delegation** - The delegate (the account being granted the permission) redeems the delegation via the Delegation Manager,
+   which verifies that the delegated authority is valid in order to perform the execution.
+
+See [how to perform executions on a user's behalf](../../guides/delegation/execute-on-users-behalf.md) to get started with the delegation lifecycle.
+
+## Delegation types
+
+You can create the following delegation types:
+
+- **Root delegation** - A root delegation is a delegation that doesn't derive its authority from another delegation.
```

Comment:
Can we make it simple, since we have not talked about re-delegation at this point, I don't expect developer to know what is deriving it's authority from another delegation. We have also not talked about authority of the delegation. 

Something like "A delegation at the top of the chain or tree is called the root delegation"?

Reply:
I added a little more clarification here.